vllm/lora/layers.py

```diff
@@ -63,6 +64,81 @@ def dec(*args, **kwargs):
     return dec
 
 
+def custom_bgmv(
```

Comment:
Have you considered to move this operation to `vllm/hpu/ops.py` since it seems to be HPU specific operation or create even create `lora_ops.py` file there? 

It is worth to consider renaming the function - the name is not self-explaining. In punica they use name _dispatch_bgmv_.

Reply:
Done. Moved ops to vllm/hpu/ops.py and renamed custom_bgmv to dispatch_bgmv_linear